### PR TITLE
fix: community page update

### PR DESCRIPTION
**Pull request by vinckr**
## Checklist

- [ ] I have read the [contributing guidelines](../blob/master/CONTRIBUTING.md) and signed the CLA.
- [ ] I have referenced an issue containing the design document if my change introduces a new feature.
- [ ] I have read the [security policy](../security/policy).
- [ ] I confirm that this pull request does not address a security vulnerability. If this pull request addresses a security
      vulnerability, I confirm that I got green light (please contact [security@ory.sh](mailto:security@ory.sh)) from the
      maintainers to push the changes.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] I have added necessary documentation within the code base (if appropriate).

### CODE REVIEW

**Comment by vinckr** [comment]
update to the community page, fyi @jfcurran @BlueAcidFrog 

I basically 
- removed a lot of fluff, 
- added links for the new example repo and archive,
- removed the listings for examples and articles

File: src/page-content/content-community.tsx [hunk 1]
```diff
@@ -21,14 +21,14 @@ export const seo: SeoProps = {
...
   id: 'community.hero',
-  title: 'Join the Ory Community',
+  title: 'Welcome to the Ory Community',
   description:
-    'You are the backbone of Ory. Members are coming from all around the world, from huge enterprises or small side projects. We may have many differences, but we all share the same goal: To build an open and free authentication and authorization standard for everyone.',
+    'The Ory community offers tutorials, examples, discussions, and answers to questions on a wide range of topics. Where to start?',
   buttons: [
     {
-      ctaTitle: 'Join the Chat on Slack',
+      ctaTitle: 'On this page we laid out some easy steps to get you started. 👇',
```
Comment (jfcurran) [inline src/page-content/content-community.tsx:29]:
```suggestion
      ctaTitle: 'Join the community',
```

File: src/page-content/content-community.tsx [hunk 2]
```diff
@@ -21,14 +21,14 @@ export const seo: SeoProps = {
 
 export const hero: HeroImageProps = {
   id: 'community.hero',
-  title: 'Join the Ory Community',
+  title: 'Welcome to the Ory Community',
   description:
-    'You are the backbone of Ory. Members are coming from all around the world, from huge enterprises or small side projects. We may have many differences, but we all share the same goal: To build an open and free authentication and authorization standard for everyone.',
+    'The Ory community offers tutorials, examples, discussions, and answers to questions on a wide range of topics. Where to start?',
```
Comment (jfcurran) [inline src/page-content/content-community.tsx:26]:
```suggestion
    'The Ory community offers tutorials, examples, discussions, and answers to questions on a wide range of topics.',
```

**Review by jfcurran — COMMENTED** [review]
How about simplifying the CTA a bit? 👇

File: src/page-content/content-community.tsx (hunk 1 above)
Comment (jfcurran) [reply src/page-content/content-community.tsx:29]: Sounds good, but keep the CTA short - 30 characters max

File: src/page-content/content-community.tsx (hunk 2 above)
Comment (jfcurran) [reply src/page-content/content-community.tsx:26]: agree, it makes sense when either the cta does a good job explaining or when the description is improved. either works 👍

**Comment by vinckr** [comment]
- Fixed the CTA according to your suggestions, thanks @jfcurran 
- Moved the duplicate entry "community" under "solutions" to "developers" and named it "github" (at it leads to our github org page) // this also is the same structure as in the footer now.

**Review by BlueAcidFrog — APPROVED** [review]
Good to go

**Pull request merged**